migrations/educandu-2022-01-07-01-create-maintenance-locks-collection.js

```diff
@@ -0,0 +1,26 @@
+// eslint-disable-next-line camelcase
+export default class Educandu_2022_01_07_01_create_maintenance_locks_collection {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async up() {
+    const existingCollections = await this.db.listCollections().toArray();
+
+    if (!existingCollections.find(x => x.name === 'maintenanceLocks')) {
```

Comment:
Not sure I get when this would be the case, that the collection is there 🤔

Reply:
It is always there, because this runs in maintenance mode, after the lock on this collection has already been taken. It's a hen/egg problem. This script basically has a semi-ceremonial effect, it does create the index though.

Reply:
Ah true, the lock is needed for the maintenance to start, but the script is run only during maintenance..
Understood.